src/js/game/LevelMVC/LevelPlane.js

```diff
@@ -87,11 +87,11 @@ module.exports = class LevelPlane extends Array {
   * Changes the block at a desired position to the desired block.
   * Important note: This is the cornerstone of block placing/destroying.
   */
-  setBlockAt(position, block) {
+  setBlockAt(position, block, force = false, skipRedstone = false) {
```

Comment:
Not used?  The `force = false` param looks like it might be the result of a bad merge.

Reply:
Please add `@param {boolean} skipRedstone Description of what the param does...` to the JSDoc comment.

Reply:
oh, yeah the force = false I don't even know if that was me? I definitely don't need it.